bitbots_body_behavior/launch/body_behavior.launch

```diff
@@ -9,5 +9,5 @@
     </group>
     <rosparam param="actionlib_client_sub_queue_size">-1</rosparam>
     <rosparam param="actionlib_server_sub_queue_size">-1</rosparam>
-    <node name="body_behavior" pkg="bitbots_body_behavior" type="body_behavior.py" respawn="true"  output="screen" />
+    <node name="body_behavior" pkg="bitbots_body_behavior" type="body_behavior.py"   output="screen" />
```

Comment:
Do we really want this? Looks dangerous.

Reply:
I do not think that this is dangerous. The code should be well enough tested to be sure that it will not die on the robot. This also makes testing easier because the nodes can be killed and restarted manually.